fish.payara.transformer.maven/src/main/java/org/eclipse/transformer/maven/TransformMojo.java

```diff
@@ -9,14 +9,15 @@
  * SPDX-License-Identifier: (EPL-2.0 OR Apache-2.0)
  ********************************************************************************/
 
+// Portions Copyright [2022] Payara Foundation and/or affiliates
```

Comment:
Portions copyright is a GPL + CDDL license thing, not required.
```suggestion
```

Reply:
I _believe_ the way this would work would be

```
Copyright (c) 2020 Contributors to the Eclipse Foundation
Copyright (c) 2022 Payara Foundation and/or its affiliates
```

Reply:
Do you mean to upgrade all the files affected after changing the module name from org.eclipse.transformer to fish.payara.transformer?

Reply:
I already added the missed copyright to the edited files